src/mwoffliner.lib.ts

```diff
@@ -380,7 +380,7 @@ async function execute(argv: any) {
     const zimCreator = new ZimCreator({
       fileName: outZim,
       fullTextIndexLanguage: dump.opts.withoutZimFullTextIndex ? '' : dump.mwMetaData.langIso3,
-      welcome: (dump.opts.mainPage ? dump.getArticleBase(dump.opts.mainPage) : 'index'),
+      welcome: (dump.opts.mainPage ? getArticleBase(dump.opts.mainPage) : 'index'),
```

Comment:
I see no reason why we should have anything URL encoded here.

Reply:
Yes, even I can't see any good reason for this to be encoded.
Done.